modules/quotas-viewing-quotas.adoc

```diff
@@ -24,10 +24,10 @@ $ oc get quota -n demoproject
 .Example output
 [source,terminal]
 ----
-NAME                AGE
-besteffort          11m
-compute-resources   2m
-core-object-counts  29m
+NAME                           AGE    REQUEST                                                                                                      LIMIT
+besteffort                     4s     pods: 1/2
+compute-resources-time-bound   10m    pods: 0/2                                                                                                    limits.cpu: 0/1, limits.memory: 0/1Gi
+core-object-counts             109s   configmaps: 2/10, persistentvolumeclaims: 1/4, replicationcontrollers: 1/20, secrets: 9/10, services: 2/10
```

Comment:
I'm assuming the output looks like this? As one long line of text for core-object-counts?

Reply:
@yep. I actually snipped it a bit to make it appear cleaner.